crates/bevy_winit/src/system.rs

```diff
@@ -101,6 +107,68 @@ pub fn create_windows<F: QueryFilter + 'static>(
     }
 }
 
+pub fn create_monitors(
```

Comment:
This also seems to update the list of known monitors (despite the name :D), maybe add this in a doc comment?

Reply:
Yeah, honestly, this should probably be renamed. `create_windows` indicates that we are spawning windows on the winit side, but in this case we are actually synchronizing in the other direction. I'll definitely add some docs to clarify at least, just thought to keep the naming the same as it's at least a conceptually similar hook between the bevy world and winit.